### PR TITLE
Ot147 34

### PR DESCRIPTION
**Pull request by DanteIturri**
add the endpoint PACH for update user for id 
 add UserDTO file and package in auth module

### CODE REVIEW

**Comment by anavduo** [comment]
I think you can re-use the UserController I added to add the method   @PatchMapping("/user/{id}")

File: src/main/java/com/alkemy/ong/auth/controller/UserAuthController.java [hunk 1]
```diff
@@ -1,16 +1,21 @@
 package com.alkemy.ong.auth.controller;
 
+import com.alkemy.ong.auth.domain.UserDomain;
+import com.alkemy.ong.auth.dto.UserDTO;
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/auth/controller/UserAuthController.java:4]: I also added a UserDTO you can import, my question is , should UserDTO be at the auth package? or in the source package? let me know what is best

Comment (DanteIturri) [reply src/main/java/com/alkemy/ong/auth/controller/UserAuthController.java:4]: I think it is indistinct but there may be other options

**Review by juan1977lopez — COMMENTED** [review]
Favor revisar dado que hay clases que dejan de existir en este PR.
Arreglar clases que se desaparecen

**Pull request closed without merging**